acm.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   acm_domains  = [for domain in var.domains : domain if domain.include_in_acm]
-  domain_parts = split(".", local.acm_domains[0].domain)
+  domain_parts = length(local.acm_domains) > 0 ? split(".", local.acm_domains[0].domain) : []
```

Comment:
@nagagovindarajan, think we don't need a condition here, was this causing any error when testing?

Reply:
@santhoshratala Yes it gave error if include_in_acm = false